src/engine/rtr.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+SRC_DIR=`realpath $(dirname "$0")`
+OUTPUT_DIR=`realpath $(mktemp -d -t wazuh-rtr-XXXXXX)`
+CONTAINER_NAME=wazuh-rtr-engine
+RTR_DIR=$SRC_DIR/.rtr
+RTR_BIN=rtr.py
+RTR_CONFIG="rtr_inputs/rtr_cicd.json"
+CURRENT_UID_GID=$(id -u -n):$(id -g -n)
+PERMISSIONS_BACKUP_FILE=/tmp/permissions.acl
+USERNAME=`logname`
+USERID=`id -u $USERNAME`
+GROUPID=`id -g $USERNAME`
+VERBOSE=''
+BUILD_DOCKER="yes"
+
+build() {
+    docker build -f .rtr/Dockerfile -t $CONTAINER_NAME -q . > /dev/null
```

Comment:
maybe it's a good enhance to strip redirect to null `(> /dev/null)` if `-v` is supplied, could see waths happens with `docker build` which usually takes a long time